index.bs

```diff
@@ -2883,7 +2949,10 @@ a [=moment=] |triggerTime|, and a [=boolean=] |fenced|:
 1. Let |aggregatableTriggerData| be the result of running [=parse aggregatable trigger data=]
     with |value|.
 1. If |aggregatableTriggerData| is null, return null.
-1. Let |aggregatableValuesConfigurations| be the result of running [=parse aggregatable values=] with |value|.
+1. Let |filteringIdsMaxBytes| be the result of [=parsing aggregatable filtering ID max bytes=] with |value|.
+1. If |filteringIdsMaxBytes| is null, return null. 
```

Comment:
Shall we validate against source registration time configuration to make sure that it’s not configured for to include if max bytes is non-default?